library/CHART-TEMPLATE/templates/config-map-of-examples.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.naming.fullname" . }}-examples
+  namespace: {{ .Release.Namespace }}
+data:
+  lowDataMode: Low data mode {{ if include "common.lowDataMode" . }}enabled{{ else }}disabled{{ end }}
+  test: {{ include "common.lowDataMode" . }}
```

Comment:
I like the idea! But perhaps we could make the cm exclusive to low data mode? That way it will appear more obviously if the test fails.

Reply:
Sure. I expanded the example.

Solved in [9b9b76e](https://github.com/newrelic/helm-charts/commit/9b9b76e3ccfbf000303049efe883be4483698ad4)